tbls/v2/tbls.go

```diff
@@ -95,3 +103,11 @@ func Verify(compressedPublicKey PublicKey, data []byte, signature Signature) err
 func Sign(privateKey PrivateKey, data []byte) (Signature, error) {
 	return impl.Sign(privateKey, data)
 }
+
+func FastAggregateVerify(shares []PublicKey, signature Signature, data []byte) error {
+	return impl.VerifyAggregate(shares, signature, data)
+}
+
+func CombineSignatures(signs []Signature) (Signature, error) {
+	return impl.Aggregate(signs)
+}
```

Comment:
Can we also rename these function too?

Reply:
Urgh, right.

The refactoring function missed this, sorry!